charts/alidns-webhook/values.yaml

```diff
@@ -47,3 +47,9 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+
+# Optional additional envs for the webhook container
+env: []
```

Comment:
I prefer to name it extraEnv as it seems to be more widely used for this use case.

Reply:
I Agree, extraEnv seems better 👌 done